app/flows/check_uk_visa_flow/outcomes/outcome_work_n.erb

```diff
@@ -62,29 +68,35 @@
   * you’ll leave at the end of your visit
   * you’re able to support yourself and your dependents during your trip (or have funding from someone else to support you)
 
-  This is the same information as someone applying for a [Standard Visitor visa](/standard-visitor-visa/eligibility) would need to provide.
+  Find out more about [visiting the UK as a Standard Visitor](/standard-visitor).
 
+  ^The rules on what you’ll need to enter the UK may be different if you’re [travelling from Ireland, Jersey, Guernsey or the Isle of Man](/guidance/travelling-between-the-uk-and-ireland-isle-of-man-guernsey-or-jersey).
+ 
   ##If you want to do paid work in the UK
 
   You must apply for a visa if you want to work:
 
-  * as a [volunteer](/temporary-worker-charity-worker-visa)
+  * as [a volunteer](/temporary-worker-charity-worker-visa)
   * in a [work experience role](/government-authorised-exchange)
-  * for a [charity](/temporary-worker-charity-worker-visa)
+  * for [a charity](/temporary-worker-charity-worker-visa)
   * for a [religious organisation](/religious-worker-visa)
   * as a [domestic worker in a private household](/domestic-workers-in-a-private-household-visa)
   * on a [transfer to your employer’s UK branch](/intracompany-transfer-worker-visa)
   * in a [skilled job](/skilled-worker-visa)
   * in [work covered by international law](/international-agreement-worker-visa), for example for a foreign government or as a private servant in a diplomatic household
   * to [provide a service on behalf of a Swiss company](/guidance/apply-for-a-service-providers-from-switzerland-visa)
-  * in [sports](/sportsperson-visa)
   * in [arts or entertainment](/creative-worker-visa)
+  * [in sports](/sportsperson-visa)
 
   ###If you want to work in arts or entertainment for 3 months or less
 
   You do not need a visa for qualifying work if you're coming to the UK for 3 months or less using the [Creative Worker visa concession](/creative-worker-visa/creative-worker-concession).
 
   You must bring a [certificate of sponsorship and evidence of savings](/creative-worker-visa/eligibility) to show officers at the UK border.
 
-  ^ You must see a Border Force officer - do not use the automatic ePassport gates.
-<% end %>
+  <% if calculator.passport_country_in_epassport_gate_list? %>
+    ^ You must see a Border Force officer - do not use the automatic ePassport gates.
+  <% end %>
+
+  <% end %>
```

Comment:
Looks like there are two `<% end %>`'s here - should one be removed?

Bit concerning if so, as the test passed regardless! 😬

Reply:
Turns out this needs to be there but wasn't indented correctly, it's the `<% end %>` for the govspeak block that starts  on line 5.

Reply:
Ah, that makes sense 🤦 I figured it was probably needed, sorry if that sent you on a wild goose chase!